defaultclient_test.go

```diff
@@ -1206,6 +1217,100 @@ func Test_GetRolePermissions(t *testing.T) {
 	}
 }
 
+func Test_GetClientInformation(t *testing.T) {
+	t.Parallel()
+
+	httpRequestTriggerCount := 0
+
+	// mock http request
+	mockHTTPClient := &httpClientMock{
+		doMock: func(req *http.Request) (*http.Response, error) {
+			httpRequestTriggerCount++
+
+			r := ioutil.NopCloser(strings.NewReader(`
+{
+   "clientID": "5a2cf6407d6349c7a75264c2c1d04a10",
```

Comment:
suggest to use random generated uuid for this

Reply:
noted